### PR TITLE
[WIP] shared storage on nfs article introduced

### PR DESCRIPTION
**Pull request by ryokugyu**

### CODE REVIEW

**Comment by ryokugyu** [comment]
Fix #103 and #429

File: static/docs/use-cases/multiple-data-scientists-on-a-single-machine.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# Shared Development Server
```
Comment (shcheklein) [inline static/docs/use-cases/multiple-data-scientists-on-a-single-machine.md]: why removing this?

File: static/docs/use-cases/shared-storage-on-nfs.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+> With large data files it is better to set the cache directory to NFS. Not only
+> just it will cache the data faster but also version the data. Thus you would
+> avoid copying large files from you NFS server to the machine.
+
+Lets configure NFS server and client to see how DVC can help to make the
+workflow faster.
+
+<details>
+
+# Setting up the NFS server
```
Comment (shcheklein) [inline static/docs/use-cases/shared-storage-on-nfs.md]: Is there a link we can share instead? Setting up NFS server/client is not DVC-specific and I would avoid including this into the doc.

File: static/docs/use-cases/shared-storage-on-nfs.md [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+get the results. It became necessary that data is accessible to everyone and
+every team member has a same updated dataset. For this example, we will be using
+NFS (Network File System) for storing and sharing files on the network.
+
+With DVC, we need not to copy the dataset on our local machine everytime when
+new data is added to dataset. We can set the `cache directory` on NFS server.
+The cached data will be present in the NFS server, which in turn will be fast to
+access and process requests faster.
+
+> With large data files it is better to set the cache directory to NFS. Not only
```
Comment (shcheklein) [inline static/docs/use-cases/shared-storage-on-nfs.md]: It should be a regular paragraph that explains _why_ it's beneficial to use DVC in this case. Namely, for people their workspace looks the same way as they are alone, DVC "magically" manages links from the workspace to cache. People can work with multiple versions of a datasets without affecting each other and without creating duplicates (like a full copy of a folder with 1M images)

File: static/docs/use-cases/shared-storage-on-nfs.md [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+> just it will cache the data faster but also version the data. Thus you would
+> avoid copying large files from you NFS server to the machine.
+
+Lets configure NFS server and client to see how DVC can help to make the
```
Comment (shcheklein) [inline static/docs/use-cases/shared-storage-on-nfs.md]: Again, not sure we need this. I would assume that it's configured already.

File: static/docs/use-cases/shared-storage-on-nfs.md [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+you and your colleagues are members of the same group (e.g. 'users') and that
+your shared directory is owned by that group and has respective permissions.
+
+Let's create a mount point of client side.
+
+```dvc
+mkdir -p /mnt/dataset/project1_data
+```
+
+### Configure Cache
```
Comment (shcheklein) [inline static/docs/use-cases/shared-storage-on-nfs.md]: we def need more explanation behind the commands below

File: static/docs/use-cases/shared-storage-on-nfs.md [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+location.
+
+> It will be beneficial to have a shared cache location on the server. It avoid
+> copying large files from shared server to the local machine.
+
+After configuring NFS on both server and client side. Let's create an export
+directory on server side where all data will be stored.
+
+```dvc
+mkdir -p /project1_data
```
Comment (shcheklein) [inline static/docs/use-cases/shared-storage-on-nfs.md]: no need to copy the name from discuss - just `/storage` should be enough

**Comment by ryokugyu** [comment]
@shcheklein please have a look at the changes and let me know any other changes required.

**Pull request closed without merging**